backend/api/views.py

```diff
@@ -1,8 +1,9 @@
 import datetime
 
 from django.db import connection
-from django.db.models import F
+from django.db.models import F, Sum
 from django.utils import timezone
+from django.db.models.functions import TruncHour
```

Comment:
:fries:

Reply:
I think we're of a dying breed that knows of potatosort.